Examples/Echo/RUNME

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+#
+# Use this to run the swift-proto generator
+#
+protoc echo.proto --swift_out=. --swift_opt=Visibility=Public --plugin=/Users/tim/bin/protoc-gen-swift
```

Comment:
> It's not that new, but it wasn't needed for 0.9.24, so grpc-swift wasn't using it.

Got it. Thanks for explaining!

> I haven't checked the podspec for this branch, so it's likely not to work without fixes. If you would advise on that, it would be great.

I will try and report back.